src/chrome/content/rules/Dailymotion.xml

```diff
@@ -1,29 +1,14 @@
 <!--
 	Nonfunctional domains:
-
+		- advertising.dailymotion.com
 		- blog.dailymotion.com
-		- press.dailymotion.com		(shows steaw.com, CN: www.steaw.com)
-		- proxy-46.dailymotion.com
-		- publicite.dailymotion.com
-		- publisher.dailymotion.com	(reset)
-		- vid.ak.dmcdn.net		(403, Akamai)
-		- vid2.ak.dmcdn.net		(504, akamai)
-
-
-	Problematic domains:
-
-		- ak2.static.dailymotion.com	(mismatched, CN: *.dmcdn.net)
-		- support.dmcloud.net		(mismatched, CN: *.zendesk.com)
-
 
 	Partially covered domains:
-
 		- (www.)dailymotion.com
-
 			- cdn/manifest/video/\w+.mnft 403s
 			- crossdomain.xml breaks videos
-
 -->
+
 <ruleset name="Dailymotion (default off)" default_off="breaks some embedded videos">
```

Comment:
The site seems to be HTTPS by default, can we enable it now?

Reply:
Removing the `default_off`. Thanks!